SPMeta2/SPMeta2.CSOM/ModelHandlers/SecurityGroupModelHandler.cs

```diff
@@ -245,6 +239,36 @@ private void DeploySiteGroup(object modelHost, SiteModelHost siteModelHost, Defi
 
             currentGroup.Update();
             context.ExecuteQueryWithTrace();
+
+            // Assign group to AssociatedXXXGroup property of web
+            // https://github.com/SubPointSolutions/spmeta2/issues/1108
+
+            if (securityGroupModel.IsAssociatedMemberGroup)
+                web.AssociatedMemberGroup = currentGroup;
+
+            if (securityGroupModel.IsAssociatedOwnerGroup)
+                web.AssociatedOwnerGroup = currentGroup;
+
+            if (securityGroupModel.IsAssociatedVisitorsGroup)
+                web.AssociatedVisitorGroup = currentGroup;
+
+            // Does this need to be called? Would it work using the securityGroupModel
```

Comment:
Could you please tell me if this is needed or if I should remove the code before you merge the PR?